test/verify/machineslib.py

```diff
@@ -2328,7 +2328,8 @@ def tryCreateThenInstall(self, dialog):
             # unfinished install script runs indefinitelly, so we need to force it off
             b.click("#vm-{0}-off-caret".format(name))
             b.click("#vm-{0}-forceOff".format(name))
-            b.wait_in_text("#vm-{0}-state".format(name), "shut off")
+            # https://bugzilla.redhat.com/show_bug.cgi?id=1818089
+            # After shutting it off virt-install will restart the domain and exit because of the above bug
```

Comment:
Could this use the same trick as the other test, and shutting it off twice? Or at least keep the original wait_in_text in a comment, so that we can update the test once the bug gets fixed?

Reply:
Shutting of twice ended up being racy - https://logs.cockpit-project.org/logs/pull-13795-20200402-134824-df461a75-debian-testing/log.html#186-2

Basically everything I tried to shut it off twice was racy:

* Waiting for `shut-off` state after first `Force off` in order to execute the second `Force off` is racy because the guest goes off only for a fraction a second
* Triggering two times shut off without checking the state both happen before the guest goes off

In any case - it's enough for only the first 'shut off' to happen in order that the second XML takes effect.